MinPlatformPkg/Acpi/AcpiTables/AcpiPlatform.c

```diff
@@ -248,7 +248,7 @@ GetTableSize (
   )
 {
   UINT32  TableLength;
-  UINT32  Index;
+  UINTN   Index;

```

Comment:
Can you add `MU_CHANGE: CodeQL` or something similar around the changes, as shown in the [contribution doc](https://microsoft.github.io/mu/CodeDevelopment/requirements/)?